### PR TITLE
chore: Add documentation on running benchmarks with Microk8s

### PR DESCRIPTION
**Pull request by andygrove**
## Which issue does this PR close?

Part of https://github.com/apache/datafusion-comet/issues/602

### CODE REVIEW

File: benchmarks/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+FROM apache/datafusion-comet:latest
+
+RUN apt update \
+    && apt install -y git python3 python3-pip \
+    && apt clean
+
+RUN cd /opt \
+    && git clone https://github.com/andygrove/datafusion-benchmarks.git \
+    && cd datafusion-benchmarks \
+    && git checkout tpcds
```
Comment (andygrove) [inline benchmarks/Dockerfile:26]: I will update this to use official repo once I have a PR merged over there

Comment (andygrove) [reply benchmarks/Dockerfile:26]: https://github.com/apache/datafusion-benchmarks/pull/12

File: benchmarks/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#
```
Comment (comphead) [inline benchmarks/Dockerfile:1]:
```suggestion
```

File: benchmarks/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    --conf spark.executor.instances=1 \
+    --conf spark.executor.memory=32G \
+    --conf spark.executor.cores=8 \
+    --conf spark.cores.max=8 \
+    --conf spark.task.cpus=1 \
+    --conf spark.executor.memoryOverhead=3G \
+    --jars local://$COMET_JAR \
+    --conf spark.executor.extraClassPath=$COMET_JAR \
+    --conf spark.driver.extraClassPath=$COMET_JAR \
+    --conf spark.plugins=org.apache.spark.CometPlugin \
```
Comment (comphead) [inline benchmarks/README.md:82]: 👍

File: benchmarks/README.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+# Location of Comet JAR within the Docker image
+export COMET_JAR=/opt/spark/jars/comet-spark-spark3.4_2.12-0.2.0-SNAPSHOT.jar
+
+$SPARK_HOME/bin/spark-submit \
+    --master k8s://https://127.0.0.1:16443 \
+    --deploy-mode cluster  \
+    --name comet-tpcbench \
+    --driver-memory 8G \
+    --conf spark.driver.memory=8G \
+    --conf spark.executor.instances=1 \
```
Comment (comphead) [inline benchmarks/README.md:73]: for benches its probably good to run it in scale? to put instances > 1

**Review by comphead — APPROVED** [review]
lgtm thanks @andygrove. some minor questions

File: benchmarks/README.md (hunk 4 above)
Comment (andygrove) [reply benchmarks/README.md:73]: This is really meant as an example. I would expect users to customize the script for their local environment.

**Review by kazuyukitanimura — APPROVED** [review]

**Comment by comphead** [comment]
@andygrove are you planning to merge it, I'd also like to add the Helm chart section to run in the Kubernetes cluster

**Pull request merged**